contrib/util/ccda_import/import_ccda.php

```diff
@@ -123,32 +128,40 @@ function outputMessage($message): void
 use OpenEMR\Common\Uuid\UuidRegistry;
 use OpenEMR\Services\Cda\CdaComponentParseHelpers;
 
+
+// clear the log so we only keep last session. Saves cleanup time.
+unlink($GLOBALS['OE_SITE_DIR'] . "/documents/temp/log.txt");
```

Comment:
here's a good place where can use the above flag to only do above in web page mode